<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrEditSavedSearchDialog</class>
 <widget class="QDialog" name="AddOrEditSavedSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="savedSearchLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Saved search</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="savedSearchNameHorizontalLayout">
     <item>
      <widget class="QLabel" name="savedSearchNameLabel">
       <property name="text">
        <string>Saved search &amp;name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>savedSearchNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="savedSearchNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="searchQueryPlainTextEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>savedSearchNameLineEdit</tabstop>
  <tabstop>searchQueryPlainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddOrEditSavedSearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddOrEditSavedSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
